### PR TITLE
importinto, disttask: allow multi import job & adjust total memory for cgroup v2

### PR DESCRIPTION
**Pull request by D3Hunter**
### What problem does this PR solve?

Issue Number: ref #49008 

Problem Summary:

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)

on a cgroup v2 32c64g node, physical mem is 62.9GiB and from cgroup we got 64GiB, but the real usable memory is 57.35GiB

before
```
[2024/03/11 04:45:31.337 +00:00] [INFO] [manager.go:94] ["build task executor manager"] [total-cpu=32] [total-mem=62.9GiB]
```
after
```
[2024/03/12 16:56:20.409 +00:00] [INFO] [manager.go:108] ["adjust memory limit for cgroup v2"] [before=62.9GiB] [after=56.32GiB]
[2024/03/12 16:56:20.409 +00:00] [INFO] [manager.go:114] ["build task executor manager"] [total-cpu=32] [total-mem=56.32GiB]
```

- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by Benjamin2037 — APPROVED** [review]
LGTM

**Comment by D3Hunter** [comment]
manually test again after code change
```
[2024/03/13 16:44:56.899 +00:00] [INFO] [manager.go:106] ["adjust memory limit for cgroup v2"] [before=62.9GiB] [after=56.32GiB]
[2024/03/13 16:44:56.899 +00:00] [INFO] [manager.go:111] ["build task executor manager"] [total-cpu=32] [total-mem=56.32GiB]
```

**Review by Benjamin2037 — APPROVED** [review]
LGTM

File: pkg/disttask/framework/taskexecutor/manager.go [hunk 1]
```diff
@@ -91,6 +92,22 @@ func NewManager(ctx context.Context, id string, taskTable TaskTable) (*Manager,
 	if totalCPU <= 0 || totalMem <= 0 {
 		return nil, errors.Errorf("invalid cpu or memory, cpu: %d, memory: %d", totalCPU, totalMem)
 	}
+	cgroupLimit, version, err := cgroup.GetCgroupMemLimit()
+	// ignore the error of cgroup.GetCgroupMemLimit, as it's not a must-success step.
+	if err == nil && version == cgroup.V2 {
```
Comment (Benjamin2037) [inline pkg/disttask/framework/taskexecutor/manager.go:97]: From the code, it seems only the Cgroup V2 return mem limit should be adjusted.

Comment (D3Hunter) [reply pkg/disttask/framework/taskexecutor/manager.go:97]: yes, v1 returns the hierarchy mem limit which is correct

**Review by lance6716 — APPROVED** [review]

**Pull request merged**